docs/reference/query-dsl/has-child-query.asciidoc

```diff
@@ -9,7 +9,7 @@ an example:
 --------------------------------------------------
 {
     "has_child" : {
-        "type" : "blog_tag",
+        "child_type" : "blog_tag",
```

Comment:
Agreed, and I'd much prefer to settle on `type` it's the option we've always documented.